hardware/products/ucm/g400.md

```diff
@@ -1,5 +1,250 @@
 # G400
 ![G400S](images/g400s.jpg) ![G400D](images/g400d.jpg)
+# Overview
+---
+The G400-D System on Module (SoM) is the precursor to the Universal Compute Modules (UCM). It pioneered the SO-DIMM System on Module (SoM) originally running .Net Micro Framework exclusively. It now can operate various other platforms, including TinyCLR.
+
+# Specifications
+---
+|                |                                              |
+| :------------- | :------------------------------------------- |
+| Processor      | AT91SAM9X35 (Atmel/Microchip 32-bit ARM 926) |
+| Speed          | 400 MHz                                      |
+| RAM            | 32 KB (SRAM)                                 |
+| Flash          | Not Supported                                |
+| External RAM   | 128 MB (DDR2 SDRAM)                          |
+| External Flash | 4 MB (SPI)                                   |
+| UART           | 1 UCM Standard (5 overall\* including HS)    |
+| UART HS        | 1 UCM Standard                               |
+| I2C            | 1 UCM Standard                               |
+| SPI            | 1 UCM Standard                               |
+| CAN            | 1 UCM Standard (2 overall\*)                 |
+| SDIO           | 1 UCM Standard                               |
+| ADC            | 8 UCM Standard                               |
+| PWM            | 4 UCM Standard                               |
+| GPIO           | 12 UCM Standard (70 overall\*)               |
+| IRQ            | 4 UCM Standard (70 overall\*)                |
+| USB Client     | Supported                                    |
+| USB Host       | Supported (2 overall\*)                      |
+| LCD            | Supported (16bpp)                            |
+| Ethernet PHY   | Supported                                    |
+| WiFi           | Not supported                                |
+| DCMI           | Not supported                                |
+| VBAT           | Supported                                    |
+| JTAG           | Full JTAG                                    |
+*\*overall means that there are more of that feature multiplexed with other pins/features*
+
+# Resources
+---
+## Documents
+* [Schematic](http://files.ghielectronics.com/downloads/Schematics/Systems/G400D_Schematic.pdf)
+
+## Design Considerations
+Under development
+
+## Module Pin Configuration  
+| SO-DIMM Pin   | Universal Compute Standard    | Function Name                                 |
+| :-----------: | :---------------------------: | :-------------------------------------------: |
+| 1             | AGND                          | GND                                           |
+| 2             | Ethernet TX-                  | ETH PHY TX-                                   |
+| 3             | Module Specific 1             | --------------------------------------------- |
```

Comment:
Use spaces instead of ---- for empty cells. It may confuse people thinking it is the enter of the header row.

Reply:
Not sure what you mean by "enter of the header row." If the dashes are not used, then I think it will look as if we forgot to fill the entry. Perhaps only a few dashes will look better if the full length is too distracting.